.github/workflows/test.yml

```diff
@@ -39,6 +39,6 @@ jobs:
 
       - run: npm install-ci-test
 
-      - run: npm run build
+      - run: npm run validate
```

Comment:
I find `validate` a bit unclear.

what about naming the two jobs `npm run test:unit` and `npm run test:acceptance`?

Then we can have an `npm run test` that runs both of them.